src/views/Home/components/GuestWelcome/GuestWelcome.module.scss

```diff
@@ -29,7 +29,7 @@ div.gw_card {
     background-image: url(./images/campus4K.jpg);
   }
 
-  @media (max-width: 1920px) {
+  @media (max-width: 1536px) {
```

Comment:
What is the size choice here? Where does this number come from?

Reply:
The [MUI breakpoint widths were changed](https://mui.com/material-ui/migration/v5-component-changes/#breakpoint-sizes). The Extra Large breakpoint was previously `1920px`, but is now `1536px`, so I found all instances of that number in our codebase and updated (along with the other changed breakpoint widths). That way, this custom CSS will stay consistent with the behavior of MUI Grid (and other breakpoint-aware components).

Reply:
Good. Any chance we can centralize this value somewhere? I just think whenever we have a constant in multiple places that will always be the same, it's good to consider only having one place where it is defined. I think you previously mentioned a hook that MUI might provide called useTheme() or something alike.

Obviously that could just be raised as an issue for now, but I wanted to ask for your thoughts.

Reply:
I definitely think that we should use the centralized reference, yes. In this case we couldn't use a hook in CSS. However, We've actually always had the centralized definitions (for both JS and CSS), people just don't always use them. I do think that that's best as a separate issue though.